test/test-lib-python.sh

```diff
@@ -40,16 +40,14 @@ function ct_pull_or_import_postgresql() {
 
 # Check the imagestream
 function test_python_imagestream() {
-  local tag="-ubi7"
-  if [ "${OS}" == "rhel8" ]; then
-    tag="-ubi8"
-  elif [ "${OS}" == "rhel9" ]; then
+  local tag="-ubi8"
+  if [ "${OS}" == "rhel9" ]; then
     tag="-ubi9"
+  elif [ "${OS}" == "rhel10" ]; then
+    tag="-ubi10"
   fi
-  if [[ "${VERSION}" == *"minimal"* ]]; then
-    VERSION=$(echo "${VERSION}" | cut -d "-" -f 1)
-  fi
-  if [[ "${VERSION}" == "3.11" ]] || [[ "${VERSION}" == "3.12" ]]; then
+  # On RHEL10 we have only 3.12-minimal image
+  if [[ "${VERSION}" == "3.11" ]] || [[ "${VERSION}" == "3.12" ]] && [[ "${VERSION}" == "3.12-minimal" ]]; then
```

Comment:
IIUC, this evaluates to: if ${VERSION} is 3.11 or if ${VERSION} is both "3.12" and "3.12-minimal", the block will be executed. Isn't the second condition always false? That would mean it will get only executed if ${VERSION} is 3.11. Is this intended? Am I misunderstanding?

Reply:
Oh, sorry, I see it's corrected in the next commit.